source/fundamentals/builders.txt

```diff
@@ -439,14 +439,17 @@ have a defined Atlas Vector Search index before you can perform a vector search
    :atlas:`Create an Atlas Vector Search Index </atlas-vector-search/create-index>` in the
    Atlas manual.
 
+.. include:: /includes/vector-search-parameters.rst
+
 Consider the ``embedded_movies`` collection in the ``sample_mflix`` database. You
 can use a ``$vectorSearch`` stage to perform a semantic search on the ``plot_embedding``
 field of the documents in the collection.
 
 The following example shows how to use builders to generate an aggregation pipeline to
 perform the following operations:
 
-- Performs a vector search on the Atlas Vector Search index of the ``plot_embedding`` field using vector embeddings for the string ``"time travel"``
+- Performs a vector search on the Atlas Vector Search index of the ``plot_embedding``
```

Comment:
It might be misleading to use `plot_embedding` here and `m => m.Embedding` in the code.
Maybe worth adding the `EmbeddedMovie` model for clarity.